src/bwamem.cpp

```
@@ -144,10 +144,13 @@ KSORT_INIT(mem_intv1, SMEM, intv_lt1)  // debug
  *   - len1 >= len2 (target >= query) : the gscore (query-end) capture is
  *     byte-identical to scalar only when the query-end column lies on/left of
  *     the main diagonal. See smithWaterman128_8 gscore capture.
- *   - w <= BSW8_MAX_W (= 127)     : every band/position quantity is encoded
- *     as a diagonal offset d = j - i in [-w, +w], which fits signed int8
- *     only for w <= 127. The default opt->w = 100 qualifies; wide-band
- *     retries (w doubling to 200/400/800) do NOT and fall back to 16-bit.
+ *   - w <= BSW8_MAX_W (= 124)     : the band/position quantities are encoded as
+ *     signed-int8 diagonal offsets spanning [-(w+1), w+3] (the band-grow term
```

Cppcheck (2.21.0)

[style] The function 'bsw_force_generic_matrix' is never used.

(unusedFunction)